tests/input/single-line-array-spacing.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+declare(strict_types=1);
+
+$singleItem = [ $emptyArray ];
+
+$multiItems = [$emptyArray,$singleItem];
```

Comment:
Does it also fix a space before the comma?
```suggestion
$multiItems = [$emptyArray ,$singleItem];
```

Reply:
No, the fix if **after** the comma:
```diff
-$multiItems = [$emptyArray,$singleItem];
+$multiItems = [$emptyArray, $singleItem];
```

AFAIK we don't have a standard for spaces between variables and before commas

Reply:
I think the sniff checks also spaces before comma.

Reply:
Indeed:

```sh
 7 | ERROR | [x] Expected 0 spaces between "$emptyArray" and comma, 1 found.
```

Reply:
Let's add it to the test then?